stylish.js

```diff
@@ -80,6 +80,9 @@ export default {
       })
     }
 
-    return `\n${underlinedFilename}\n${table(output)}`
+    if (underlinedFilename) {
+      return `\n${underlinedFilename}\n${table(output)}`
+    }
+    return `\n${table(output)}`
```

Comment:
Could you write a separate test on this logic? Just that number of lines are 1 less, or something like that :smile: